src/stores/EndpointConfig/Store.ts

```diff
@@ -245,6 +245,28 @@ const getInitialState = (
                     endpointSchema,
                 } = get();
 
+                // if (
+                //     entityType === 'capture' &&
+                //     Object.hasOwn(data[0].spec, 'autoDiscover') &&
+                //     isObject(data[0].spec.autoDiscover)
+                // ) {
+                //     const autoDiscoverySettings = data[0].spec.autoDiscover;
+
+                //     setAddNewBindings(
+                //         Object.hasOwn(
+                //             autoDiscoverySettings,
+                //             'addNewBindings'
+                //         ) && autoDiscoverySettings.addNewBindings
+                //     );
+
+                //     setEvolveIncompatibleCollections(
+                //         Object.hasOwn(
+                //             autoDiscoverySettings,
+                //             'evolveIncompatibleCollections'
+                //         ) && autoDiscoverySettings.evolveIncompatibleCollections
+                //     );
+                // }
```

Comment:
Please add a comment explaining the plan for this code

Reply:
No, it clearly needs to be removed.